src/libtriton/includes/triton/bitsVector.hpp

```diff
@@ -73,7 +73,7 @@ namespace triton {
         TRITON_EXPORT void setLow(triton::uint32 v);
 
         //! Sets the pair<high, low> position
-        TRITON_EXPORT void setPair(std::pair<triton::uint32, triton::uint32> p);
+        void setPair(triton::uint32 high, triton::uint32 low);
```

Comment:
I think we should find a new name now :D. What about:

* setBit(s)Position
* setSize
* setBit(s)
* ?

Reply:
I suggest setValue. It seems to be most close.

Reply:
Modified the message.